### PR TITLE
merge_sort.kt

### PR DESCRIPTION
**Pull request by Awani-Shukla**
**Fixes issue:**

**Changes:** this file is of merge sort in kotlin

### CODE REVIEW

File: code/sorting/src/merge_sort/merge_sort.kt [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import java.util.*
```
Comment (arnavb) [inline code/sorting/src/merge_sort/merge_sort.kt]: Add newline below.

File: code/sorting/src/merge_sort/merge_sort.kt [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+                temp = left.removeAt(0)
+            }
+            else {
+                temp = right.removeAt(0)
+            }
+            merged.add(temp)
+        }
+        if (!left.isEmpty()) merged.addAll(left)
+        if (!right.isEmpty()) merged.addAll(right)
+         return merged
```
Comment (arnavb) [inline code/sorting/src/merge_sort/merge_sort.kt]: Indentation.

Comment (Awani-Shukla) [reply code/sorting/src/merge_sort/merge_sort.kt]: @arnavb please check your mail.

Comment (arnavb) [reply code/sorting/src/merge_sort/merge_sort.kt]: @Awani-Shukla I will unfortunately be unable to accept your offer, as I have many academic demands (I am a mere high school student). Sorry!

Comment (Awani-Shukla) [reply code/sorting/src/merge_sort/merge_sort.kt]: no problem.

**Comment by Awani-Shukla** [comment]
@arnavb  any more changes?

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Comment by danos7777** [comment]
I'm new to this and I really like someone that's got time to help me with things on my phone that has time when I get time .thank you for your email I hope someone can help.

**Pull request merged**